server/lib/base-logger.ts

```diff
@@ -33,11 +33,15 @@ export class BaseLogger {
   wazuhLogger: winston.Logger | undefined = undefined;
   wazuhPlainLogger: winston.Logger | undefined = undefined;
   PLAIN_LOGS_PATH: string = '';
+  PLAIN_LOGS_FILE_NAME: string = '';
   RAW_LOGS_PATH: string = '';
+  RAW_LOGS_FILE_NAME: string = '';
 
-  constructor(plainLogsPath: string, rawLogsPath: string) {
-    this.PLAIN_LOGS_PATH = plainLogsPath;
-    this.RAW_LOGS_PATH = rawLogsPath;
+  constructor(plainLogsFile: string, rawLogsFile: string) {
+    this.PLAIN_LOGS_PATH = path.join(WAZUH_DATA_LOGS_DIRECTORY_PATH, plainLogsFile);
+    this.RAW_LOGS_PATH = path.join(WAZUH_DATA_LOGS_DIRECTORY_PATH, rawLogsFile);
```

Comment:
Could we move the variable that stores the path of these files to the `constants.ts` file?